backend/graminsta/post/services.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: UTF-8 -*-
+"""services.py"""
+
+from django.contrib.auth import get_user_model
+from django.shortcuts import get_object_or_404
+from core.serializers import UserSerializer
+from .models import FollowRelationship
+
+
+def get_people_user_follows(username):
+    """
+    Returns users that the given user follows.
+
+    Parameters
+    ------------
+    username: str
+
+    Returns
+    -------
+    follow_people: The QuerySet contains users that the
+        given user follows.
+    """
+    user = get_user_model().objects.get(username=username)
+    ids = FollowRelationship.objects.filter(from_user=user) \
+        .values_list('to_user_id', flat=True)
+    queryset = get_user_model().objects.filter(id__in=ids)
+    following_people = UserSerializer(queryset, many=True)
```

Comment:
As commented in @shiyi001 's PR, for APIs in `services.py`, you usually need to return the original `models.Model` object, instead of return the serialized representations of them, since you might have other calls to other APIs that requires output of this call. So in your case, return `queryset` directly.

```
return get_user_model().objects.filter(id__in=ids)
```